eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubSnpNoHgvsToDbsnpVariantsWrapperProcessorTest.java

```diff
@@ -79,7 +73,7 @@ public static void tearDownClass() throws Exception {
     }
 
     @Test
-    public void processSubmittedVariantAllelesMismatch() throws Exception {
+    public void processSubSnpNoHgvsToWrapper() throws Exception {
```

Comment:
For the sake of completion, could you implement another test with multiple submitted variants?